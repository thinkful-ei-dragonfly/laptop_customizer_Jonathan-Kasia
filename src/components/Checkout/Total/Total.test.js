import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import Total from './Total'

const selected = {
  Processor: {
      name: 'MacBookPro',
      cost: 30
    },
  "Operating System": {
      name: 'Ubuntu Linux 16.04',
      cost: 45
    },
  "Video Card":{
      name: 'NVidia',
      cost: 500
    },
  Display: {
      name: 'Sony',
      cost: 750
    }
}

describe('Total component', () => {
  it('Renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<Total selected={selected} />, div)
    ReactDOM.unmountComponentAtNode(div)
  })

  it('Renders the UI as expected', () =>{
    const tree = renderer
    .create(<Total selected={selected}  />)
    .toJSON();
    expect(tree).toMatchSnapshot();
  })
})