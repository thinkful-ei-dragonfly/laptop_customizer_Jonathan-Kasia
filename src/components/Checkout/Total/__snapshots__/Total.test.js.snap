// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Total component Renders the UI as expected 1`] = `
<div
  className="summary__total"
>
  <div
    className="summary__total__label"
  >
    Your Price: 
  </div>
  <div
    className="summary__total__value"
  >
    $1,325.00
  </div>
</div>
`;
